package/yast2-cluster.spec

```diff
@@ -1,7 +1,7 @@
 #
 # spec file for package yast2-cluster
 #
-# Copyright (c) 2018 SUSE LINUX GmbH, Nuernberg, Germany.
+# Copyright (c) 2019 SUSE LLC
```

Comment:
I see you are regularly changing this line in the file (you seem the only one who cares about it :wink:). Just a question out of curiosity. Shouldn't one keep the original year while changing the copyright note? I mean, should this read as this?

```
Copyright (c) 2014-2019 SUSE LLC
```

Just asking, I'm not an expert.

Reply:
I get the Copyright change information of "SUSE LLC" via [SUSE open source policy](https://opensource.suse.com/suse-open-source-policy).

IIRC, a latest `osc` will help on converting the Copyright when submit, but it is not helping in Yast's case, which included by source package.
I agree `2014-2019` makes more sense, but i just inherit the format since long long ago:(